cime/config/e3sm/config_grids.xml

```diff
@@ -1,1262 +1,1611 @@
+      <grid name="glc">null</grid>
+      <grid name="wav">null</grid>
+      <mask>gx1v6</mask>
+    </model_grid>
+
+    <model_grid alias="conusx4v1_conusx4v1" compset="(DOCN|XOCN|SOCN|AQP1)">
+      <grid name="atm">ne0np4conusx4v1lowcon</grid>
+      <grid name="lnd">ne0np4conusx4v1lowcon</grid>
+      <grid name="ocnice">ne0np4conusx4v1lowcon</grid>
+      <grid name="rof">null</grid>
+      <grid name="glc">null</grid>
+      <grid name="wav">null</grid>
+      <mask>tx0.1v2</mask>
+    </model_grid>
+
+    <model_grid alias="svalbardx8v1_svalbardx8v1" compset="(DOCN|XOCN|SOCN|AQP1)">
+      <grid name="atm">ne0np4svalbardx8v1lowcon</grid>
+      <grid name="lnd">ne0np4svalbardx8v1lowcon</grid>
+      <grid name="ocnice">ne0np4svalbardx8v1lowcon</grid>
+      <grid name="rof">null</grid>
+      <grid name="glc">null</grid>
+      <grid name="wav">null</grid>
+      <mask>tx0.1v2</mask>
+    </model_grid>
+
+    <model_grid alias="sooberingoax4x8v1_sooberingoax4x8v1" compset="(DOCN|XOCN|SOCN|AQP1)">
+      <grid name="atm">ne0np4sooberingoax4x8v1lowcon</grid>
+      <grid name="lnd">ne0np4sooberingoax4x8v1lowcon</grid>
+      <grid name="ocnice">ne0np4sooberingoax4x8v1lowcon</grid>
+      <grid name="rof">null</grid>
+      <grid name="glc">null</grid>
+      <grid name="wav">null</grid>
+      <mask>tx0.1v2</mask>
+    </model_grid>
+
+    <model_grid alias="enax4v1_enax4v1" compset="(DOCN|XOCN|SOCN|AQP1)">
+      <grid name="atm">ne0np4enax4v1</grid>
+      <grid name="lnd">ne0np4enax4v1</grid>
+      <grid name="ocnice">ne0np4enax4v1</grid>
+      <grid name="rof">null</grid>
+      <grid name="glc">null</grid>
+      <grid name="wav">null</grid>
+      <mask>oRRS18to6</mask>
+    </model_grid>
+
+    <model_grid alias="enax4v1_ne30_enax4v1" compset="(DOCN|XOCN|SOCN|AQP1)">
+      <grid name="atm">ne0np4enax4v1</grid>
+      <grid name="lnd">ne30np4</grid>
+      <grid name="ocnice">ne0np4enax4v1</grid>
+      <grid name="rof">null</grid>
+      <grid name="glc">null</grid>
+      <grid name="wav">null</grid>
+      <mask>oRRS18to6</mask>
+    </model_grid>
+
+    <model_grid alias="ne16_ne16" compset="(DOCN|XOCN|SOCN|AQP1)">
+      <grid name="atm">ne16np4</grid>
+      <grid name="lnd">ne16np4</grid>
+      <grid name="ocnice">ne16np4</grid>
+      <grid name="rof">r05</grid>
+      <grid name="glc">null</grid>
+      <grid name="wav">null</grid>
+      <mask>gx3v7</mask>
+    </model_grid>
+
+    <model_grid alias="ne30_ne30" compset="(DOCN|XOCN|SOCN|AQP1)">
+      <grid name="atm">ne30np4</grid>
+      <grid name="lnd">ne30np4</grid>
+      <grid name="ocnice">ne30np4</grid>
+      <grid name="rof">r05</grid>
+      <grid name="glc">null</grid>
+      <grid name="wav">null</grid>
+      <mask>gx1v6</mask>
+    </model_grid>
+
+    <model_grid alias="ne30_oEC_ICG" compset="(CAM5.+CLM.+MPASO%SPUNUP)">
+      <grid name="atm">ne30np4</grid>
+      <grid name="lnd">ne30np4</grid>
+      <grid name="ocnice">oEC60to30ICG</grid>
```

Comment:
The ocnice grid name has been oEC60to30_ICG - I think leaving the underscore off will cause this to fail unless we make other changes to match

Reply:
Nice catch, I think there may have been a bug in my converter script.